index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <ul>
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
```

Comment:
- [ ] The hamburger button should be inside a proper `<button>` tag so in the future you can add a javascript event to make the mobile menu section open through this element. Kindly change that.
 - [ ] Your hamburger button is smaller than the Figma template. Try increasing its size to match the template.